packages/lib/passport-interface/src/genericIssuanceTypes.ts

```diff
@@ -1,3 +1,4 @@
+import { getErrorMessage } from "@pcd/util";
```

Comment:
Agreed - it's there out of convenience, and because `genericIssuanceTypes.ts` was there already, but it would make more sense to create a separate package for PODBox-specific code. I'll separate it out.